### ISSUE TITLE
[Platform] GET certificates request is very heavyweight for the platform

### ISSUE THREAD

**Issue by SergeyPotachev** [yugabyte/yugabyte-db#9404]
There are 55 universes and 461 certificates in DB. The CertificateController.list() operation takes a lot of time for the execution, so the API request fails with error code 504 "Gateway Time-out".

The actual reason is likely to be the fact that for each certificate we are calling CertificateInfo.getInUse() which calls Universe.universeDetailsIfCertsExists() which in turn gets all the existing universes each time with the universe details filling.
This needs to be reworked - we can read all the universes only once, build the table in memory and then prepare the result list.

Treating this as a bug because currently the functionality is broken.

**Comment by SergeyPotachev** [comment]
Here is a set of serial stack dumps from the affected platform:

[stack.txt](https://github.com/yugabyte/yugabyte-db/files/6856344/stack.txt)
[stack2.txt](https://github.com/yugabyte/yugabyte-db/files/6856345/stack2.txt)
[stack3.txt](https://github.com/yugabyte/yugabyte-db/files/6856346/stack3.txt)
[stack4.txt](https://github.com/yugabyte/yugabyte-db/files/6856347/stack4.txt)

**Comment by SergeyPotachev** [comment]
Duplicate of #9365.
